packages/react/src/components/TimePicker/TimePickerDropdown.test.jsx

```diff
@@ -197,7 +197,6 @@ describe('TimePickerDropdown', () => {
     expect(qbt('time-picker-test-spinner')).toBeInTheDocument();
     userEvent.click(document.body);
     await waitFor(() => expect(qbt('time-picker-test-spinner')).not.toBeInTheDocument());
-    expect(screen.queryByText(/The time entered is invalid/)).toBeTruthy();
```

Comment:
This should not be removed. If all these steps in this test occur, this assertion should be truthy.